src/Microsoft.ML.OnnxTransformer/DnnImageFeaturizerTransform.cs

```diff
@@ -56,6 +54,26 @@ internal DnnImageFeaturizerInput(string outputColumnName, string inputColumnName
     /// | Input column data type | Vector of <xref:System.Single> |
     /// | Output column data type | Vector of <xref:System.Single>, the size of the vector depends on the pre-trained DNN |
     ///
+    /// NuGet requirements:
+    ///	- Microsoft.ML.OnnxTransformer
+    /// - Microsoft.ML.OnnxRuntime(question: does OnnxTransformer already pull OnnxRuntime as dependency? if so we don't need to add this)
+    /// - Microsoft.ML.OnnxRuntime.Gpu(only if GPU processing is used)
+    /// - Each pre-trained DNN model has a separate NuGet that must be included if that model is used:
+    ///     - Microsoft.ML.DnnImageFeaturizer.AlexNet
```

Comment:
>     [](start = 8, length = 4)

let's use 2-spaces instead of 4, for the next level indentation. 4 might make the parser confuse it with code block.